### ISSUE TITLE
stylus button action not working properly with new input system

### ISSUE THREAD

**Issue by suisseWalter** [xournalpp/xournalpp#2713]
(Please complete the following information, and then delete this line)

**Affects versions :**
 - OS: Ubuntu 20:04
 - (Linux only) Desktop environment: I3
 - Which version of libgtk do you use
     - 3.24.20
 - Version of Xournal++:
     1.0.20-debhotfix
 - Installation method: [e.g. PPA, Flatpak, official release, etc.]
      manual install of the .deb package.
 - Device:
      HP-Elitebook-x360 with HP active pen 2.
**Describe the bug**
the Stylus buttons do not properly work with the NEW input system. with the old input system I could set one button to do erasing and one to use the selection tool. 
with the new input system one button is always a eraser while the other one does nothing. even if I put every button to  the selection tool I can't use it to select anything. I believe this problem also exists in some other (older ) versions, I haven't tested all of them. but at some point in the past the new input system worked properly. 

**To Reproduce**
1. turn on new input system(and restart )
2. set anything but eraser as stylus button (and "eraser" button) action.
3. see that it still erases with one button and does nothing with the others.

**Expected behavior**
I can set one button the eraser and one to the selection tool and they work this way. 

**Additional context**
if needed I can also test on gnome and gnome-wayland if the problems exists there to. 
also I expect this problem to exists on most HP devices with this pen(and maybe also some lenovo as the pens are 100% compatible).
I'm mainly reporting this as 1.0.20 says in the notes that it will be the last release with the old input system.

**Comment by rolandlo** [comment]
Please try out version 1.1.0+dev and test if the new input system works to your expectations. You can check it out with the AppImage from the [nightly releases](https://github.com/xournalpp/xournalpp/releases/tag/nightly)

**Comment by suisseWalter** [comment]
I got it to work with the experimental version. 
steps to fix:
assign the input devices differently(correctly) for me the eraser was mapped as pen while the pen was mapped as mouse and I had to manually map them to what they where(pen as pen, eraser as eraser). The buttons are now inverted to what they where with the old input system(who cares about that) 
also worth a note, the new dark mode works properly in 1.1.0dev.
